JBBCode/CodeDefinition.php

```diff
@@ -11,28 +11,28 @@
  */
 class CodeDefinition
 {
-    /* NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
+    /* @var string NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
```

Comment:
Huh, I thought this changed in PSR 5; it appears I was wrong. Going to change this as soon as I get home.